Resources/Private/Fusion/Documents/Default/Default.fusion

```diff
@@ -0,0 +1,44 @@
+prototype(Neos.Demo:Default) < prototype(Fusion:Template) {
```

Comment:
I don't feel good about inheriting such huge pile of body code...
In the case of `Neos.Demo:Default.Document` with all of its stylesheets and stuff it's a lesser evil, but for content it would be annoying that you'd always need to look back and forth to see what is defined at this page, what's available and what's not...
I would extract repetitive stuff like menus into a layout object, and compose it rather than inherit it.